### PR TITLE
Fix for #21

### PR DESCRIPTION
**Pull request by voodoofrog**
I've added two settings to the main config for old style attribute tabbing, and old style HP tabbing. I can condense this down to one setting if preferred.

Because the original sheets were using different controls they used `type=hidden` but because this sheet uses more semantically appropriate controls, I've instead removed them from the tab order via `tabindex=-1`.

### CODE REVIEW

**Review by kgar — CHANGES_REQUESTED** [review]
I've got a few minor requests regarding the wording of the localization keys / values and the setting variable names.
All in all, this looks great!

**Comment by kgar** [comment]
I apologize for the wait on the review. I put comments in like 3 days ago, but I didn't know I needed to complete the review for them to post. I'll get that right next time.

File: public/lang/en.json [hunk 1]
```diff
@@ -251,6 +251,14 @@
     "name": "Item Card Delay (ms)",
     "hint": "Delays showing item cards for the specified milliseconds (ms). 0 means no delay. Default: 300."
   },
+  "T5EK.Settings.UseOldAttributeTabbing": {
+    "name": "Use old attribute keyboard tabbing",
```
[the remark this answers is not in the document]
Comment (voodoofrog) [reply public/lang/en.json:255]: Absolutely.

**Comment by voodoofrog** [comment]
> I apologize for the wait on the review. I put comments in like 3 days ago, but I didn't know I needed to complete the review for them to post. I'll get that right next time.

Yeah no worries. The review process in GitHub is a bit different from what I'm used to too, I usually work with GitLab.

**Comment by voodoofrog** [comment]
I've updated with changes to bring it in line with your requests.

**Review by kgar — APPROVED** [review]
Fantastic work. I will be using these settings, myself.

Thank you for contributing!

**Pull request merged**